rego/Calendar.rego

```diff
@@ -87,40 +87,14 @@ if {
 #
 # Baseline GWS.CALENDAR.1.2v0.1
 #--
-ExtSharingSecondaryCalSettingDetailsStr(LastEvent) := Description if {
-    LastEvent.NewValue == "SHOW_ONLY_FREE_BUSY_INFORMATION"
-    Description := concat("", [
-        "<span class=setting>Only free busy/information for secondary calendars </span> is shared outside ",
-        LastEvent.DomainName
-    ])
-}
 
-ExtSharingSecondaryCalSettingDetailsStr(LastEvent) := Description if {
-    LastEvent.NewValue == "READ_ONLY_ACCESS"
-    Description := concat("", [
-        "<span class=setting>All information for secondary calendars </span> is shared outside ",
-        LastEvent.DomainName,
-        " but outsiders cannot change calendars."
-    ])
-}
-
-ExtSharingSecondaryCalSettingDetailsStr(LastEvent) := Description if {
-    LastEvent.NewValue == "READ_WRITE_ACCESS"
-    Description := concat("", [
-        "<span class=setting>All information for secondary calendars </span> is shared outside ",
-        LastEvent.DomainName,
-        " and outsiders can change calendars."
-    ])
-}
-
-ExtSharingSecondaryCalSettingDetailsStr(LastEvent) := Description if {
-    LastEvent.NewValue == "MANAGE_ACCESS"
-    Description := concat("", [
-        "<span class=setting>All information for secondary calendars </span> is shared outside ",
-        LastEvent.DomainName,
-        " and outsiders can manage calendars"
-    ])
-}
+GetFriendlyValue1_2(Value) := "Secondary Calendars Share all information, but outsiders cannot change calendars." if {
+    Value == "READ_ONLY_ACCESS"
+} else := "Share all information, and outsiders can change calendars." if {
+    Value == "READ_WRITE_ACCESS"
+} else := "Share all information, and allow managing of calendars." if {
+    Value == "MANAGE_ACCESS"
+} else := "Only free/busy information (hide event details)."
```

Comment:
Suggest adding a condition to handle the "SHOW_ONLY_FREE_BUSY_INFORMATION" setting name so we handle all cases, then have the last else default to the `Value` parameter. 

Example from chat: 

```
#
# Baseline GWS.CHAT.2.1v0.1
#--
GetFriendlyValue2_1(Value) := "Allow all files" if {
    Value == "ALL_FILES"
} else := "Images only" if {
    Value == "IMAGES_ONLY"
} else := Value
```

Reply:
Just pushed those changes through